apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -146,19 +156,19 @@ const TutorialDetail = React.createClass({
                 <div style={{clear: 'both'}}/>
                 <table style={styles.tutorialDetailsTable}>
                   <tbody>
-                  {this.props.item.teachers_notes &&
-                    <tr key={0}>
-                      <td style={styles.tutorialDetailsTableTitle}>
-                        More resources
-                      </td>
-                      <td style={styles.tutorialDetailsTableBody}>
-                        <a href={this.props.item.teachers_notes} target="_blank">
-                          <i className="fa fa-external-link" aria-hidden={true}></i>
-                          &nbsp;
-                          Teachers' notes
-                        </a>
-                      </td>
-                    </tr>}
+                    {this.props.item.teachers_notes &&
+                      <tr key={0}>
+                        <td style={styles.tutorialDetailsTableTitle}>
+                          More resources
+                        </td>
+                        <td style={styles.tutorialDetailsTableBody}>
+                          <a href={this.props.item.teachers_notes} target="_blank">
+                            <i className="fa fa-external-link" aria-hidden={true}></i>
+                            &nbsp;
+                            Teachers' notes
+                          </a>
+                        </td>
+                      </tr>}
```

Comment:
Can you add another row below Teacher Notes? "Short link: https://hourofcode.com/shortcode" that only shows for activities tagged with `online-tutorial` for activity type?

![image](https://cloud.githubusercontent.com/assets/1920530/19785671/25e302c6-9c4f-11e6-9387-aed6325b5100.png)